_data/master/navbars/getting-started.yml

```diff
@@ -25,6 +25,8 @@ toc:
       path: /getting-started/kubernetes/installation/config-options
     - title: Integration guide (advanced) 
       path: /getting-started/kubernetes/installation/integration
+    - title: Installing Calico with Application Layer Policy
```

Comment:
I would bump this up a bit in the tree to put it just under Installing Calico for policy (advanced). I am not sure, but is this feature going to be beta or something like that? If so, you could add in parentheses `(beta)`

Reply:
Perhaps could change to "Enabling application layer policy"